.github/workflows/docs.yml

```diff
@@ -26,10 +27,16 @@ jobs:
           flutter pub global run dartdoc
           ls -al doc/api
 
-      - uses: ably/sdk-upload-action@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: eu-west-2
+          role-to-assume: arn:aws:iam::884096606738:role/ably-sdk-builds-ably-flutter
+          role-session-name: "${{ github.run_id }}-${{ github.run_number }}"
+
+      - name: Upload Documentation
+        uses: ably/sdk-upload-action@v1.2.2
```

Comment:
Any reason to lock in to a patch version here? Surely just leaving it at v1 will give the same behaviour?